### PR TITLE
Added build script for react-scheduler

### PR DESCRIPTION
**Pull request by nailushapotnuru**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: r/react-scheduler/react_scheduler_ubi8.5.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#! /bin/bash
```
Comment (seth-priya) [inline r/react-scheduler/react_scheduler_ubi8.5.sh:1]: this should be "#/bin/bash -e"?

File: r/react-scheduler/react_scheduler_ubi8.5.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+PACKAGE_VERSION=$(jq -r ".version" package.json)
+# run the test command from test.sh
+if ! npm install && npm audit fix && npm audit fix --force; then
+        echo "------------------$PACKAGE_NAME:install_fails-------------------------------------"
+        echo "$PACKAGE_URL $PACKAGE_NAME" 
+        echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_Fails" 
+        exit 1
+fi
+
+# Tests are N/A.
```
Comment (seth-priya) [inline r/react-scheduler/react_scheduler_ubi8.5.sh:53]: just confirming - does this imply there are no automated tests available for the package?

**Review by seth-priya — CHANGES_REQUESTED** [review]

File: r/react-scheduler/react_scheduler_ubi8.5.sh (hunk 1 above)
Comment (nailushapotnuru) [reply r/react-scheduler/react_scheduler_ubi8.5.sh]: Done

File: r/react-scheduler/react_scheduler_ubi8.5.sh (hunk 2 above)
Comment (nailushapotnuru) [reply r/react-scheduler/react_scheduler_ubi8.5.sh:53]: Yes Priya, there are no tests available in the package.

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]

File: r/react-scheduler/react_scheduler_ubi8.5.sh (hunk 2 above)
Comment (nailushapotnuru) [reply r/react-scheduler/react_scheduler_ubi8.5.sh:54]: Downgraded the node version, build is now success.

**Review by kandarpamalipeddi — APPROVED** [review]

**Comment by kandarpamalipeddi** [comment]
Closing this PR as @bhagat-singh001 is going to work on it.

**Pull request closed without merging**